src/pages/Vaults/Vault/UpdateCollateralModal/index.tsx

```diff
@@ -10,14 +10,13 @@ import clsx from 'clsx';
 import { roundTwoDecimals, newMonetaryAmount, CollateralUnit, CurrencyUnit } from '@interlay/interbtc-api';
```

Comment:
What are these changes for? Wasn't expecting any changes in the application in this PR.

Reply:
FYI: https://github.com/interlay/interbtc-ui/pull/367#discussion_r913291984